src/Components/HaveRead/HaveRead.test.js

```diff
@@ -2,9 +2,20 @@ import React from 'react';
 import HaveRead from './HaveRead';
 import { render, screen } from '@testing-library/react';
 import '@testing-library/jest-dom';
+import { combineReducers, createStore } from 'redux';
```

Comment:
Great start here with a unit test. We will have to look into testing the overall integrated functionality with Redux